hedera-mirror-rest/controllers/contractController.js

```diff
@@ -966,7 +967,7 @@ class ContractController extends BaseController {
     const ethTransaction = ethTransactions[0];
 
     let fileData = null;
-    if (ethTransaction && !_.isNil(ethTransaction.callDataId)) {
+    if (ethTransaction && !_.isNil(ethTransaction.callDataId) && isValidUserFileId(ethTransaction.callDataId)) {
```

Comment:
simplify this line the other same line to
```suggestion
    if (utils.isValidUserFileId(ethTransaction?.callDataId)) {
```